pkg/hive/manager.go

```diff
@@ -237,3 +239,27 @@ func (hr *clusterManager) ResetCorrelationData(ctx context.Context, doc *api.Ope
 		return hr.hiveClientset.Update(ctx, cd)
 	})
 }
+
+func (hr *clusterManager) installLogsForDeployment(ctx context.Context, cd *hivev1.ClusterDeployment) (*string, error) {
```

Comment:
nit: might be good to rename this to `installLogsForLatestDeployment`, I think we've talked about the fact that this relies on our current architecture/assumption of only 1 provision existing? This would make it more clear for future us to see.